api/clients/node_client.go

```diff
@@ -8,7 +8,6 @@ import (
 	grpcnode "github.com/Layr-Labs/eigenda/api/grpc/node"
 	"github.com/Layr-Labs/eigenda/core"
 	"github.com/Layr-Labs/eigenda/encoding"
-	"github.com/Layr-Labs/eigenda/node"
```

Comment:
moved some methods from `node/utils.go` to `core/serialization.go` to prevent this client from importing `node` package